.isort.cfg

```diff
@@ -1,3 +1,6 @@
 [settings]
-force_single_line=True
```

Comment:
will this also impact the rules around only importing one object per line? Although I'm not particularly fond of the precedent, I think that choice should be discussed in a separate PR.

Reply:
I don't see a way to get `black` and `isort` to work together with `force-single-line`.

Starting from:

```python
from an.absurdly.long.path.that.forces.this.line.over.the.textwidth import module_a
from an.absurdly.long.path.that.forces.this.line.over.the.textwidth import module_b
from an.absurdly.long.path.that.forces.this.line.over.the.textwidth import module_c
```

`isort` will break lines with backslashes using the old config:

```python
from an.absurdly.long.path.that.forces.this.line.over.the.textwidth import \
    module_a
from an.absurdly.long.path.that.forces.this.line.over.the.textwidth import \
    module_b
from an.absurdly.long.path.that.forces.this.line.over.the.textwidth import \
    module_c
```

but then `black` will replace those with parens:

```python
from an.absurdly.long.path.that.forces.this.line.over.the.textwidth import (
    module_a,
)
from an.absurdly.long.path.that.forces.this.line.over.the.textwidth import (
    module_b,
)
from an.absurdly.long.path.that.forces.this.line.over.the.textwidth import (
    module_c,
)
```

...which `isort` will turn back into slashes, and etc.

I don't see an `isort` option that uses parens but doesn't group imports, so if we want to use `black` this may be the best we can do:

```python
from an.absurdly.long.path.that.forces.this.line.over.the.textwidth import (
    module_a,
    module_b,
    module_c,
)
```